Add - spandsp a library needed to setup fax capabilities

fritzkink opened the pull request.

fritzkink committed:
Add - spandsp a library needed to setup fax capabilities

AndWac commented on components/multimedia/spandsp/Makefile:
```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Friedrich Kink
+
+BUILD_BITS=	64
+
```
BUILD_BITS= 64 is our default now an can be omitted.

AndWac commented on components/multimedia/spandsp/Makefile:
```diff
@@ -0,0 +1,67 @@
+COMPONENT_LICENSE_FILE= COPYING
+
+TEST_TARGET=	$(NO_TESTS)
+
```
There seem to be tests in the sources. Have you tried to run them?

AndWac commented on components/multimedia/spandsp/Makefile:
```diff
@@ -0,0 +1,67 @@
+
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_PRE_CONFIGURE_ACTION  = ( \
```
Have you tried to use our preferred jpeg implementation? You just need to add# build with the distribution preferred libjpeg implementation
CFLAGS   += $(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
CXXFLAGS += $(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
LDFLAGS  += $(JPEG_LDFLAGS)
CONFIGURE_ENV += CPPFLAGS="$(JPEG_CPPFLAGS)"

here.

fritzkink commented:
Ok, I removed the 64bit setting and also changed the jpeg libs to the preferred ones (thanks for the hint). 
With respect to the tests - those need extra packages like netpbm:
'configure: error: "Cannot make tests without pbmtog3 installed (does your system require a netpbm-progs package?)"
make: *** [/usr/share/src/oi-userland/make-rules/configure.mk:166: /usr/src/oi-userland/components/multimedia/spandsp/build/amd64/.configured] Error 1
'
which I didn't want to create. Is it ok?
